ivy/container/general.py

```diff
@@ -3596,3 +3596,153 @@ def supports_inplace_updates(
             prune_unapplied=prune_unapplied,
             map_sequences=map_sequences,
         )
+
+    @staticmethod
+    def static_get_num_dims(
+        x: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+        /,
+        *,
+        as_array: bool = False,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+    ) -> ivy.Container:
+        """
+        ivy.Container instance method variant of ivy.get_num_dims. This method
+        simply wraps the function, and so the docstring for ivy.get_num_dims
+        also applies to this method with minimal changes.
+
+        Parameters
+        ----------
+        x
+            ivy.Container to infer the number of dimensions for
+        as_array
+            Whether to return the shape as a array, default False.
+        key_chains
+            The key-chains to apply or not apply the method to. Default is None.
+        to_apply
+            If True, the method will be applied to key_chains, otherwise key_chains
+            will be skipped. Default is True.
+        prune_unapplied
+            Whether to prune key_chains for which the function was not applied.
+            Default is False.
+        map_sequences
+            Whether to also map method to sequences (lists, tuples). Default is False.
+        
+
+        Returns
+        -------
+        ret
+            Shape of the array
+
+        Examples
```

Comment:
the static method examples should show the usage of `static_get_num_dims()` and not `ivy.get_num_dims()`

Reply:
changed as instructed